infrastructure/template.yaml

```diff
@@ -144,7 +144,7 @@ Resources:
                   ImposterApiUrl: !ImportValue third-party-stubs-ImposterStubApiUrl,
                 }
           - Fn::If:
-              - IsStagingEnvironment
+              - IsStagingOrIntegrationEnvironment
```

Comment:
@surajkumar can we streamline this condition

```
      Value:
        Fn::If:
          - Fn::If:
              - IsStagingOrIntegrationEnvironment
              - "https://test-api.service.hmrc.gov.uk/individuals/authentication/authenticator/api/match"
              - Fn::If:
                  - IsProductionEnvironment
                  - "https://api.service.hmrc.gov.uk/individuals/authentication/authenticator/api/match"
                  - Fn::Sub:
                        - "${ImposterApiUrl}/individuals/authentication/authenticator/api/match"
                        - ImposterApiUrl: !ImportValue third-party-stubs-ImposterStubApiUrl
                          
```

Reply:
Yes that's a good suggestion